File: python/Iniciante/URI1015.py
from math import sqrt

linha_1 = input().split()
x1, y1 = list(map(float, linha_1))

linha_2 = input().split()
x2, y2 = list(map(float, linha_2))

distancia = sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)

print(f"{distancia:.4f}")

File: java/iniciante/URI1015.java
package URI;

import java.util.Scanner;
import java.text.DecimalFormat;


public class URI1015 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        DecimalFormat decimal = new DecimalFormat("0.####");

        double x1 = scanner.nextDouble(), y1 = scanner.nextDouble();
        double x2 = scanner.nextDouble(), y2 = scanner.nextDouble();


        double dist = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

        String distancia = decimal.format(dist);
        System.out.println(distancia);
    }
}
